src/status_update.js

```diff
@@ -0,0 +1,15 @@
+//Update Local storage with True or False
+const updateLocal = () => {
+  const localData = JSON.parse(localStorage.getItem('List'));
+  const todos = document.querySelectorAll(".description");
+  for (let i=0; i<todos.length; i++) {
+    if(todos[i].classList.contains('check-to-do')) {
+      localData[i].complited = true;
+    } else {
+      localData[i].complited = false;
+    }
+  }
+  localStorage.setItem('List',JSON.stringify(localData));
+}
```

Comment:
# Optional Changes
Good job working on the changes so far. 
- [ ] I noticed that when I marked a todo as completed, it gets marked and the actions also reflect on the localStorage. But when I refresh the page, the todos marked as completed are not fetched and displayed on the screen as it's in LocalStorage. It is good that when we make a todo as completed, it gets stored to localStorage and when the page refreshes, the same todo should be marked as completed. The previous reviewer did not spot that, please work on that. I trust you can. 👍